nipype/utils/nipype_cmd.py

```diff
@@ -0,0 +1,69 @@
+import os
+import argparse
+import inspect
+import sys
+from nipype.interfaces.base import Interface
+
+
+def listClasses(module=None):
+    if module:
+        __import__(module)
+        pkg = sys.modules[module]
+        print "Available Interfaces:"
+        for k,v in pkg.__dict__.items():
+            if inspect.isclass(v) and issubclass(v, Interface):
+                print "\t%s"%k
+
+def add_options(parser=None, module=None, function=None):
+    interface = None
+    if parser and module and function:
+        __import__(module)
+        interface = getattr(sys.modules[module],function)()
+
+        for k,v in interface.inputs.items():
+            if hasattr(v, "mandatory") and v.mandatory:
+                parser.add_argument(k, help=v.desc)
+            else:
+                parser.add_argument("--%s"%k, dest=k,
```

Comment:
do you want to add type information here from traits? then that will help reduce the code in run_instance